portals/publisher/src/main/webapp/source/src/app/components/Apis/Details/Scopes/EditScope.jsx

```diff
@@ -166,20 +164,28 @@ class EditScope extends React.Component {
         this.handleInputs = this.handleInputs.bind(this);
         this.handleRoleDeletion = this.handleRoleDeletion.bind(this);
         this.handleRoleAddition = this.handleRoleAddition.bind(this);
-        this.validateScopeDescription = this.validateScopeDescription.bind(this);
-        this.validateScopeDisplayName = this.validateScopeDisplayName.bind(this);
+        this.validateScopeDescription =
```

Comment:
change not required. Shall we check all such places in the 3 files you have changed and remove all unnecessary changes?